badges.go

```diff
@@ -145,4 +145,12 @@ var BADGES = []Badge{
 			return c
 		},
 	},
+	{
+		"first-merge",
+		"First merge",
+		"Has at least one Pull Request merged",
```

Comment:
Description suggestion: "Got his first merge request merged"

Reply:
I kept `Pull Request` instead of `merge request`.

Reply:
Ho yes sorry. I'm using gitlab at work so I got use to "Merge Request" 😄